gframe/duelclient.cpp

```diff
@@ -1397,8 +1397,9 @@ int DuelClient::ClientAnalyze(char * msg, unsigned int len) {
 	}
 	case MSG_SELECT_UNSELECT_CARD: {
 		/*int selecting_player = */BufferIO::ReadInt8(pbuf);
-		bool buttonok = !!BufferIO::ReadInt8(pbuf);
-		mainGame->dField.select_cancelable = BufferIO::ReadInt8(pbuf);
+		bool finishable = BufferIO::ReadInt8(pbuf);
+		bool cancelable = BufferIO::ReadInt8(pbuf);
```

Comment:
this will give out a warning for converting int into bool, why not `!!BufferIO::ReadInt8(pbuf);` ?